copy.go

```diff
@@ -43,22 +47,20 @@ func switchboard(src, dest string, info os.FileInfo, opt Options) (err error) {
 	default:
 		err = fcopy(src, dest, info, opt)
 	}
-
 	return err
 }
 
-// copy decide if this src should be copied or not.
+// copySkipping decide if this src should be copied or not.
```

Comment:
Do we really need to change the name of this internal method? and the name `copySkipping` doesn't make sense to me. I prefer to keep just `copy` here. Let me know your thought.

Reply:
I only changed it, because `copy` itself is a go-keyword and if possible should not be used to prevent any confusion. Would you have a better name for that function, then I'd change it accordingly.

Reply:
I reverted this to `copy` but you really should think about a different name to avoid confusions.

Reply:
Makes sense. Let me think.

Reply:
I'm gonna change the name after I merge your PR. Please just leave my comment here ;)